### ISSUE TITLE
list out of range

### ISSUE THREAD

**Issue by liang0112-ne** [princeton-nlp/WebShop#25]
我通过webshop搭建环境，在运行bolaa时，出现下面的错误：
127.0.0.1 - - [05/Sep/2023 15:12:46] "GET /fixed_23 HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/liang/miniconda/envs/webshop/lib/python3.8/site-packages/flask/app.py", line 2095, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/liang/miniconda/envs/webshop/lib/python3.8/site-packages/flask/app.py", line 2080, in wsgi_app
    response = self.handle_exception(e)
  File "/home/liang/miniconda/envs/webshop/lib/python3.8/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/liang/miniconda/envs/webshop/lib/python3.8/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/liang/miniconda/envs/webshop/lib/python3.8/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/liang/miniconda/envs/webshop/lib/python3.8/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/mnt/e/liang/codes/task1QA/26_Bolaa/WebShop/web_agent_site/app.py", line 71, in index
    goal = goals[goal_dix]
    查看信息是没有生成足够多的fixed_id.json文件，请问这与我加载数据集少有关系吗？我使用的是数量为1000的数据集，而不是全部。

**Comment by zhilizju** [comment]
Have you resolved it? I am encountering the same issue as you. Is it because of the 'small' mode?

**Comment by ethanluoyc** [comment]
Switching to the all dataset would fix this problem.

**Comment by ysymyth** [comment]
hi, maybe try to switch to all dataset as @ethanluoyc suggested? I think it should solve the issue

**Comment by ysymyth** [comment]
close it for now but feel free to open a new one

**Comment by ZHUWEI-hub** [comment]
Hello, I am also trying to reproduce this series of papers. In the webshop environment, my fixed_0-fixed_100 is different from the instructions in React and Reflexion papers. I noticed that their instructions in fixed_0-fixed_100 are the same.
I have used my webshop/ setup.sh -d all 
Because when I use/ When setting up. sh to - d small, accessing fixed_index will list out of range
Looking forward to your reply, thank you
